core/services/run_manager.go

```diff
@@ -89,7 +89,7 @@ func newRun(
 	run.CreationHeight = models.NewBig(currentHeight)
 	run.ObservedHeight = models.NewBig(currentHeight)
 
-	if !MeetsMinimumPayment(job.MinPayment, payment) {
+	if !MeetsMinimumPayment(&job.MinPayment, payment) {
```

Comment:
In line with the previous comment, should we just pass the value?

Reply:
I'm sort of following the convention of passing large values by pointer here.

Reply:
Ok, up to you. `MeetsMinimumPayment` is only called twice, always passing `job.MinPayment` and it's entirely inside this file. In the fight against nil reference exceptions, I was shooting for a win.